lib/ptabler/schema/src/read_frame.ts

```diff
@@ -0,0 +1,20 @@
+import type { PTableDef, PObjectId } from '@milaboratories/pl-model-common';
+
+/**
+ * Represents the configuration for a step that reads data from a PFrame directory into the tablespace.
+ * Creates a lazy table from the provided PFrame using polars-pf.
+ */
+export interface ReadFrameStep {
+  /** The type of the step, which is always 'read_frame' for this operation. */
+  type: 'read_frame';
+  /** The name assigned to the loaded DataFrame in the tablespace. */
+  name: string;
+  /** Request to create the table from the PFrame. */
+  request: PTableDef<PObjectId>;
+  /** Translation of PFrame column ids into Polars column names. */
+  translation: Record<string, string>;
```

Comment:
Need more documentation here, how it is connected with file names, which part of the mapping is what.